scripts/main.js

```diff
@@ -39,16 +39,24 @@ Hooks.once('socketlib.ready', () => {
  */
 Hooks.once('ready', async () => {
   const settings = new Settings();
-
-  if (!settings.customEffectsItemId) {
+  const effectHelpers = new EffectHelpers();
+  let { customEffectsItemId } = settings;
+  if (!customEffectsItemId) {

```

Comment:
Can probably just revert this to `settings.customEffectsItemId`

Reply:
Reverted